product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -26,26 +26,16 @@ def setUpClass(cls):
                     'name': cls.supplier1.id,
                     'min_qty': 5,
                     'price': 50,
+                    'sequence': 1,
```

Comment:
Why do you need to add this sequence?

Reply:
In order to have an explicit order of `seller_ids` items

Reply:
As the price used is taken from the first supplierinfo found in `_get_supplierinfo_pricelist_price`

Reply:
But if you add the lines on UI, do they have different sequence?

Reply:
They do: `<field name="sequence" widget="handle"/>`